source/features/comment-fields-keyboard-shortcuts.tsx

```diff
@@ -39,7 +40,9 @@ function eventHandler(event: delegate.Event<KeyboardEvent, HTMLTextAreaElement>)
 			});
 
 		if (lastOwnComment) {
-			select<HTMLButtonElement>('.js-comment-edit-button', lastOwnComment)!.click();
+			// Make edit comment button appear
+			select('details[id]', lastOwnComment)!.dispatchEvent(new MouseEvent('mouseover'));
+			(await elementReady('.js-comment-edit-button', {target: lastOwnComment, stopOnDomReady: false}) as HTMLButtonElement).click();
```

Comment:
No need for that specific button. GitHub will react to any click on `<button class="js-comment-edit-button"/>`
```suggestion
			// Make the comment editable (the native edit button might not be available yet)
			const editButton = <button class="js-comment-edit-button" hidden/>;
			lastOwnComment.append(editButton);
			editButton.click();
			editButton.remove();
```

Reply:
The same way, you can create a whole new button for `edit-comments-faster`

Reply:
Is this confirmed to work? I didn't directly test my code

Reply:
Your code works very well (instant repsonse) 👍